### DISCUSSION TITLE
Default interval of 1s not respected

### DISCUSSION THREAD

**Discussion by RahulGautamSingh** [Hexagon/croner#268] [Q&A] [answered]
`* 1 * * *`

When using the above schedule and trying to get the `nextRun` or `nextRuns(10)`. The runs are scheduled after a minute and not after `1s`. Is this a bug or expected? 

To reproduce you can execute this snippet:

```js
import { Cron } from "croner";

const job = new Cron("* 21 * * *");
console.log(job.nextRuns(10));
```

**Comment by Hexagon** [comment, marked as answer]
Hello 👋 

When using a five part pattern like `* 21 * * *`, the first part represents minutes. So this is every minute, hour 21, and so on... Seconds is defaulted to 0 (just like system cron).

What you probably want is a six part pattern `* * 21 * * *`, where the first part indicates seconds.
